app/client/src/plugins/Linting/utils/getLintingErrors.ts

```
@@ -71,25 +75,88 @@ function getEvaluationScriptPosition(scriptType: EvaluationScriptType) {
+      clearTimeout: "readonly",
+      console: "readonly",
+    };
+
+    for (const dataKey in data) {
+      globalData[dataKey] = "readonly";
+
+      const dataValue = data[dataKey] as DataTreeEntity;
+
+      if (
+        !!dataValue &&
+        dataValue.hasOwnProperty("ENTITY_TYPE") &&
```

Biome (1.9.4)

[error] Do not access Object.prototype method 'hasOwnProperty' from target object.

It's recommended using Object.hasOwn() instead of using Object.hasOwnProperty().
See MDN web docs for more details.

(lint/suspicious/noPrototypeBuiltins)